e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.cs

```diff
@@ -120,9 +120,13 @@ internal static class TestHelper {
         /// </summary>
         /// <param name="simulatedOpcServer">Dictionary with URL of PLC-PLC as key and Content of Published Nodes files as value</param>
         /// <param name="context">Shared Context for E2E testing Industrial IoT Platform</param>
-        public static void SavePublishedNodesFile(PublishedNodesEntryModel simulatedOpcServer, IIoTPlatformTestContext context) {
+        public static void SavePublishedNodesFile(PublishedNodesEntryModel simulatedOpcServer, int numberOfNodes, IIoTPlatformTestContext context) {
```

Comment:
>int numberOfNodes, [](start = 95, length = 18)

Logic how many (and which nodes) should be published, should be within the test case not the test helper. The Test helper should only serialize the PublishedNodesEntryModel

Reply:
moved